UI/Content View/ContentView.swift

```diff
@@ -0,0 +1,284 @@
+                        shortcutsSection
+                            .padding(.vertical, isCompactWidth ? 16 : 24)
+                    }
+                    .padding(isCompactWidth ? 12 : 24)
+                }
+            }
+        }
+        .background(
+            GeometryReader { geometry in
+                Color.clear.preference(key: ViewWidthKey.self, value: geometry.size.width)
+            }
+        )
+        .onPreferenceChange(ViewWidthKey.self) { width in
+            self.viewWidth = width
+        }
+        .frame(minWidth: 500, minHeight: isCompactWidth ? 800 : 680)
+        .preferredColorScheme(.dark)
+    }
+}
+
+private extension ContentView {
+    var titleSection: some View {
+        VStack(spacing: 12) {
+            Text("WindowWeaver")
+                .font(.custom("SF Pro Display", size: isCompactWidth ? 32 : 40, relativeTo: .title))
+                .fontWeight(.bold)
+                .foregroundStyle(
+                    LinearGradient(
+                        colors: [Color.accent, Color.secondaryAccent],
+                        startPoint: .leading,
+                        endPoint: .trailing
+                    )
+                )
+
+            Text("Advanced Window Management")
+                .font(.custom("SF Pro Text", size: isCompactWidth ? 16 : 18))
+                .foregroundColor(.textSecondary)
+        }
+    }
+
+    var compactLayout: some View {
+        VStack(spacing: 20) {
+            // Corner Controls
+            sectionTitle("Corner Controls")
+            Grid(horizontalSpacing: 12, verticalSpacing: 12) {
+                GridRow {
+                    button(for: .corner(.topLeft))
+                    button(for: .corner(.topRight))
+                }
+                GridRow {
+                    button(for: .corner(.bottomLeft))
+                    button(for: .corner(.bottomRight))
+                }
+            }
+
+            sectionTitle("Vertical Split")
+            HStack(spacing: 12) {
+                button(for: .vertical(.top))
+                button(for: .vertical(.bottom))
+            }
+
+            sectionTitle("2/3 Split")
+            HStack(spacing: 12) {
+                button(for: .twoThirds(.left))
+                button(for: .twoThirds(.right))
+            }
+
+            sectionTitle("Third Split")
+            VStack(spacing: 12) {
+                button(for: .third(.left))
+                button(for: .third(.center))
+                button(for: .third(.right))
+            }
+
+            sectionTitle("Half Split")
+            VStack(spacing: 12) {
+                button(for: .half(.left))
+                button(for: .half(.right))
+            }
+
+            sectionTitle("Other")
+            VStack(spacing: 12) {
+                button(for: .generic(.center))
+                button(for: .generic(.fullScreen))
+            }
+        }
+    }
+
+    var regularLayout: some View {
```

Comment:
There are still ways to reduce the duplication here.

The only variable here left is the spacing in-between.